operations/flags.go

```diff
@@ -86,7 +86,7 @@ func addLargeFlag(flags ...cli.Flag) []cli.Flag {
 func addTasksFlag(flags ...cli.Flag) []cli.Flag {
 	return append(flags, cli.StringSliceFlag{
 		Name:  joinFlagNames(tasksFlagName, "t"),
-		Usage: "task name",
+		Usage: "task name(s)",
```

Comment:
No need for a fake plural - just make it plural

Reply:
I removed this and the one from the variants flag for consistency.